okapi-ir/src/test/scala/org/opencypher/okapi/ir/impl/IrBuilderTest.scala

```diff
@@ -913,11 +913,10 @@ class IrBuilderTest extends IrTestSuite {
           |  SET a :Label
           |RETURN GRAPH""".stripMargin
 
-      intercept[UnsupportedOperationException](query.asCypherQuery().model)
+      intercept[UnTypedExpr](query.asCypherQuery().model) //todo: also correct to throw UnTypedExpr instead of UnsupportedOperationException?
```

Comment:
I think this exception is perfectly acceptable. It means that it looks for the type of `a` but can not find it, which seems correct.
```suggestion
      intercept[UnTypedExpr](query.asCypherQuery().model)
```